### DISCUSSION TITLE
Release 1.2.0 Beta 14

### DISCUSSION THREAD

**Discussion by apohl79** [apohl79/audiogridder#1040] [General]
## PLUGIN
- Improvements to the plugin monitor
- Stability fixes

## SERVER
- Performance improvements
- Screen recording tweaks
- **NEW** Added mouse offset correction support
- Always add the default layout to plugins with no layout match
- Removed scan timeout warning popup and improved timeout handling
- Fixed an issue where a server with a new ID was inheriting the default UUID 
- Stability fixes

<hr /><em>This discussion was created from the release <a href='https://github.com/apohl79/audiogridder/releases/tag/'>Release 1.2.0 Beta 14</a>.</em>

**Comment by Pabloscotty** [comment]
Thank you Andreas

**Comment by giacometti777** [comment]
I just want to thank you for your high aspirations and hard work, Andreas! I have been on 1.1 stable release because I could never get a 1.2 beta release functioning completely. This one seems to be a winner (fingers crossed). It seems much more efficient than 1.1 and just as stable (if not more). It is amazing world changing work you are doing here!

**Comment by fojerhar** [comment]
Hi Andreas
Nice job again, works really well.
I have not found, what does this fixed size outbound buffer makes?
When should it be used?
If it is on, It makes the M1 Mac crazy (buffer errors, destroyed audio) with protools, but I have not found any difference with my hack at home.

thank you

**Reply by apohl79** [reply]
Hey Viktor,

the fixed buffer sitting works as follows: if the DAW initializes a plugin, is reports a sample block size. But it might pass smaller sample blocks than reported. If this setting is enabled, AG will fill blocks with the reported size before sending a block to the server. Without that setting, data gets transferred immediately. So the latency should be better if this is deactivated, but it is on per default, as this was the behavior in 1.1.

Regarding the audio issues you had, it might be related to a big I fixed a few days back, that distorted the audio when the DAW was actually sending smaller blocks.

Andreas

**Comment by fojerhar** [comment]
Hi Andreas

Just found an issue, that if a plugin is bypassed in the chain, the delay compensation falls apart at b14.
![Screen Shot 2022-10-13 at 13 49 03](https://user-images.githubusercontent.com/72139943/195588773-0cac71d6-75c2-4242-94d9-6d792bd4de43.png)
started this session with b9 I think, and reopened with every release, the plug was bypassed all the time, and just now realized that this causes this issue
Apart from this, everything is nice
Thank you
Viktor

**Reply by apohl79** [reply]
Created a bug issue. Thx!
